task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawerComponents/TaskDescriptionEditor.vue

```diff
@@ -129,7 +132,15 @@ export default {
     });
   },
   methods: {
+    disableApply (val){
+      this.buttonOff = !(val<= this.MESSAGE_MAX_LENGTH);
+    },
     saveDescription: function (newValue) {
+      let pureText = newValue ? newValue.replace(/<[^>]*>/g, '').replace(/&nbsp;/g, '').trim() : '';
```

Comment:
This completly remove the text formatting
For example, if I add new lines in the description, and save, new lines are lost.
We need to keep the html markup and count it in the character counter